pkg/metrics/metrics.go

```diff
@@ -0,0 +1,49 @@
+package metrics
+
+import (
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type serverMetrics struct {
+	/****Metrics related to Ark server backup***********/
+	lastBackupStatus prometheus.Gauge
+}
+
+var metrics *serverMetrics
```

Comment:
I'd like to avoid the package-level variable. Would you mind looking at https://github.com/heptio/gimbal/blob/master/discovery/pkg/metrics/metrics.go and basing this file on that one?

Reply:
that's helpful!